terraform/application/config/sandbox.tfvars.json

```diff
@@ -1,8 +1,9 @@
 {
     "cluster": "production",
     "namespace": "bat-production",
+    "enable_monitoring" : true,
     "statuscake_alerts": {
-        "website_url": [ "https://sandbox.claim-funding-for-mentor-training.education.gov.uk/healthcheck/all" ],
+        "website_url": [ "https://sandbox.claim-funding-for-mentor-training.education.gov.uk/healthcheck/all","https://manage-school-placements.education.gov.uk/healthcheck/all" ],
         "contact_groups": [311842]
     }
 }
```

Comment:
Currently sandbox and staging do not work as enable_monitoring: true is not set. Underlying monitor group resources will also need to be created.

Reply:
"enable_monitoring" : true added tfvars